Automattic/jetpack
# Add Survey Dialog on Plugins Deactivate

#### Changes proposed in this Pull Request:

* Load a script with Jetpack so that we can intercept clicks on the plugins screen `/wp-admin/plugins.php`
* Show Jetpack Disconnect Modal warning window when user try to deactivate Jetpack

**Is this a new feature or does it add/remove features to an existing part of Jetpack?**
* Internal reference: parBdM-hs-p2

#### Testing instructions:
1. navigate to `/wp-admin/plugins.php` with Jetpack Active and connected
2. click "deactivate" for Jetpack
<img width="650" alt="Screen Shot 2020-03-12 at 3 31 03 PM" src="https://user-images.githubusercontent.com/2810519/76572335-fbb3bb00-6476-11ea-871d-dac744b8900a.png">
3. Verify the dialog works as it does on the dashboard
<img width="650" alt="Screen Shot 2020-03-12 at 3 31 34 PM" src="https://user-images.githubusercontent.com/2810519/76572452-5d742500-6477-11ea-8fb9-4ce80d48e177.png">

4. Verify that click "Cancel" cancels the deactivation

5. Verify that after clicking "Disconnect and Deactivate" you are taken back to the plugins screen with Jetpack deactivated.

6. Connect Jetpack again and visit Jetpack Dashboard

7. Click Manage Connection and check that the modal is still working as expected there

#### Proposed changelog entry for your changes:

* Add Jetpack Disconnect Modal to WordPress Plugins screen deactivate link
